lib/language_pack/ruby.rb

```diff
@@ -117,6 +118,66 @@ def compile
     raise e
   end
 
+  def patch_libpq
+    # Check for existing libraries
+    out = run!("ls /usr/lib/x86_64-linux-gnu/ | grep libpq")
```

Comment:
This can be simplified to `run!("ls /usr/lib/x86_64-linux-gnu/libpq.so.5.11")` and catching the error, since we only want to apply the logic if 5.11 is there.